fl4health/utils/losses.py

```diff
@@ -216,22 +301,20 @@ def clear(self) -> None:
         """
         self.losses_list = []
 
-    def compute(self) -> Losses:
-        """
-        Compute sum of current list of losses if non-empty.
 
-        Returns:
-            Losses: New Losses object with sum of losses in losses_list.
-        """
-        assert len(self.losses_list) > 0
-
-        # Compute average checkpoint and backward losses across list
-        checkpoint_loss = torch.sum(torch.FloatTensor([losses.checkpoint for losses in self.losses_list]))
-        backward_loss = LossMeter.aggregate_loss_by_type(
-            self.losses_list, loss_type=LossType.BACKWARD, loss_meter_type=LossMeterType.ACCUMULATION
-        )
-        additional_loss = LossMeter.aggregate_loss_by_type(
-            self.losses_list, loss_type=LossType.ADDITIONAL, loss_meter_type=LossMeterType.ACCUMULATION
-        )
-        losses = Losses(backward=backward_loss, checkpoint=checkpoint_loss, additional_losses=additional_loss)
-        return losses
+def _aggregate_losses_dict(
```

Comment:
I'm not 100% sure on the "proper" design, so please disagree if the following seems silly. However, since this is working specifically on "Loss Dictionaries", should we make this a `@staticmethod` within the Losses class rather than a "file-wide" function?

Reply:
Yes, that's a better idea. Will do.